### PR TITLE
Get pre-release browser favicons from favicons folder

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

- updates for beta, developer, and nightly
- delete old favicons

## Issue / Bugzilla link

#7915

## Testing

### CODE REVIEW

File: bedrock/firefox/templates/firefox/all.html [hunk 1]
```diff
@@ -75,6 +93,8 @@
 {% block site_header_logo %}
   {% if platform == 'desktop' and channel == 'alpha' %}
     <h2><a href="{{ url('firefox.developer.index') }}">{{ high_res_img('logos/firefox/developer-wordmark-dark.png', {'alt': _('Firefox Developer Edition'), 'width': '260', 'height': '48'}) }}</a></h2>
+  {% elif platform == 'desktop' and channel == 'beta' %}
+    <h2>{{ high_res_img('/protocol/img/logos/firefox/browser/beta/logo-word-hor-md.png', {'alt': 'Firefox Beta', 'width': '260', 'height': '48'}) }}</h2>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/all.html]: This is throwing an error [on demo](https://bedrock-demos-pr-7920.herokuapp.com/en-US/firefox/beta/all/): `ValueError: Missing staticfiles manifest entry for '/protocol/img/logos/firefox/browser/beta/logo-word-hor-md.png'`

See https://github.com/mozilla/bedrock/issues/7863

File: bedrock/firefox/templates/firefox/releases/nightly-feed.xml [hunk 2]
```diff
@@ -12,10 +12,10 @@
   <id>{{ url('firefox.nightly.notes.feed')|absolute_url }}</id>
   <link rel="self" type="application/atom+xml" href="{{ url('firefox.nightly.notes.feed')|absolute_url }}"/>
   <link rel="alternate" type="text/html" href="{{ url('firefox.notes', channel='nightly')|absolute_url }}"/>
-  <icon>{{ static('img/firefox/nightly/favicon.png')|absolute_url }}</icon>
+  <icon>{{ static('img/favicons/firefox/browser/nightly/apple-touch-icon.png')|absolute_url }}</icon>
   <updated>{{ notes[0].modified.isoformat() }}</updated>
   <webfeeds:cover image="{{ static('img/firefox/nightly/page-image.png')|absolute_url }}" />
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/releases/nightly-feed.xml]: Could we change this to use the Nightly page image that's in Protocol?

**Comment by stephaniehobson** [comment]
Updated.

Integration tests running here: https://ci.vpn1.moz.works/blue/organizations/jenkins/bedrock_multibranch_pipeline/detail/run-integration-tests/393/pipeline

**Comment by stephaniehobson** [comment]
Hm, the review fixes aren't showing up here.

**Comment by stephaniehobson** [comment]
There we go.

**Comment by stephaniehobson** [comment]
Updated tests are running here: https://ci.vpn1.moz.works/blue/organizations/jenkins/bedrock_multibranch_pipeline/detail/run-integration-tests/404/pipeline/

**Comment by stephaniehobson** [comment]
Rebased on #7935 and tests are passing.

**Pull request merged**